01-HTML-basics/exercise1_html/table.html

```diff
@@ -3,6 +3,9 @@
 	<head>
   		<meta name="description" content="Table of expenses">
  		<meta name="author" content="Ezequiel Vannucci">
+ 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+ 		<title>Table</title>
 	</head>
 
 
```

Comment:
If you want to add a `th` to the table is good to also add a `thead` and a `tbody` for the table body